stacks/PickupStack.ts

```diff
@@ -30,11 +34,35 @@ export function PickupStack ({ stack }: StackContext): void {
     // for debug!
     enableExecuteCommand: true
   })
+  service.taskDefinition.taskRole.addManagedPolicy(ManagedPolicy.fromAwsManagedPolicyName('SecretsManagerReadWrite'))
+  // configure the custom image to log router
+  service.taskDefinition.addFirelensLogRouter('log-router',{
+    firelensConfig: {
+      type: FirelensLogRouterType.FLUENTBIT,
+    },
+    image: ContainerImage.fromRegistry('grafana/fluent-bit-plugin-loki:1.6.0-amd64'),
+  })
+
+  const grafanasecret = aws_secretsmanager.Secret.fromSecretNameV2(
+    stack,
+    'gf-id',
+    'grafanahost',
+  );
```

Comment:
We will want to make exporting logs to loki conditional per environment.  For example, PR based envs should not be sending that info over.

Reply:
Now the job name is conditional, based on the env name, like staging-pickup, prod-pickup or pr58-pickp (this pr)

![image](https://user-images.githubusercontent.com/15051788/210598150-a050f18e-a354-4291-a61c-9141bd8cc8c4.png)